Traceback (most recent call last):
  File "/home/navu/python-code/trace.py", line 3, in <module>
    raise Exception('This is the error message.')
Exception: This is the error message.
